### PR TITLE
Proxy PR for Long Context Capability 1513

### PR DESCRIPTION
**Pull request by gagb**
## Why are these changes needed?
This is a new PR in lieu of #1513 because that one was made from fork and @sonichi wanted it to be autogen. 
So I created a new branch in microsoft/autogen, merged my fork into, and created a new PR 

## Related issue number

#1513 

## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

**Review by rickyloynd-microsoft — APPROVED** [review]
Looking forward to it!

**Comment by sonichi** [comment]
image
https://github.com/microsoft/autogen/actions/runs/7823032387/workflow
Could you please fix the workflow file?

**Comment by sonichi** [comment]
You can change the workflow back after the tests passed. Thanks.

**Comment by gagb** [comment]
> You can change the workflow back after the tests passed. Thanks.

It passed. I updated from main before trying to change to pull_request_target. Now it's again waiting for approval. Should I have not done that. I'd love to understand what's going on here in the next SCRUM.

**Review by ekzhu — APPROVED** [review]

File: .github/workflows/contrib-openai.yml
Comment (sonichi) [inline .github/workflows/contrib-openai.yml:7]:
```suggestion
  pull_request_target:
```

**Comment by sonichi** [comment]
> > You can change the workflow back after the tests passed. Thanks.
> 
> It passed. I updated from main before trying to change to pull_request_target. Now it's again waiting for approval. Should I have not done that. I'd love to understand what's going on here in the next SCRUM.

You can ask authors of other contrib-openai tests. They've all gone through this.

**Comment by rickyloynd-microsoft** [comment]
> > > You can change the workflow back after the tests passed. Thanks.
> > 
> > 
> > It passed. I updated from main before trying to change to pull_request_target. Now it's again waiting for approval. Should I have not done that. I'd love to understand what's going on here in the next SCRUM.
> 
> You can ask authors of other contrib-openai tests. They've all gone through this.

I've gone through this process, but I don't remember the details. We should really document it in a prominent place.

**Comment by sonichi** [comment]
> > > > You can change the workflow back after the tests passed. Thanks.
> > > 
> > > 
> > > It passed. I updated from main before trying to change to pull_request_target. Now it's again waiting for approval. Should I have not done that. I'd love to understand what's going on here in the next SCRUM.
> > 
> > 
> > You can ask authors of other contrib-openai tests. They've all gone through this.
> 
> I've gone through this process, but I don't remember the details. We should really document it in a prominent place.

Or find a better solution.

**Comment by gagb** [comment]
relevant to #1478

**Pull request merged**
